drivers/bridge/bridge.go

```diff
@@ -1077,6 +1105,24 @@ func (d *driver) DeleteEndpoint(nid, eid string) error {
 		}
 	}()
 
+	// Try removal of neighbor proxy. Discard error: it is a best effort.
+	// Also make sure defer does not see this error either.
+	if n.config.NDPProxyInterface != "" && n.config.EnableIPv6 {
+		link, err := d.nlh.LinkByName(n.config.NDPProxyInterface)
+		if err == nil {
```

Comment:
:+1: 

Does this makes sense ?
```
logrus.Warnf("could not remove ndp neighbour, because could not find link for ndp proxy interface %q: %v", config.NDPProxyInterface, err)
```